credoai/artifacts.py

```diff
@@ -462,7 +402,7 @@ def predict_proba(X):
 class CredoData:
     """Class wrapper around data-to-be-assessed
 
-    CredoData serves as an adapter between tabular datasets
+    CredoData serves as an adapter between datasets
```

Comment:
Can we add a couple of lines here or at the bottom of the Docstring, specifying what
the expected types for X/y/sensitive features are after the whole validation + processing is completed?

Reply:
Sure